ddtrace/llmobs/_experiment.py

```diff
@@ -38,6 +41,29 @@ class DatasetRecord(TypedDict):
     record_id: NotRequired[Optional[str]]
 
 
+class _TaskResult(TypedDict):
```

Comment:
since we're in an internal module i don't think we need to underscore prefix this guy